MobiusCore/Source/Mobius.swift

```diff
@@ -154,13 +154,13 @@ public extension Mobius {
         /// Create a `MobiusController` from the builder
         ///
         /// - Parameters:
-        ///   - defaultModel: The initial default model of the `MobiusController`
+        ///   - initialModel: The initial default model of the `MobiusController`
```

Comment:
Should we remove the “default” here in the comment as well?

Reply:
It is the “initial default model” of the controller, since a new default model can be set with `replaceModel`.